src/hooks/integration/useGetIntegrations.tsx

```diff
@@ -0,0 +1,42 @@
+import { useQuery } from "@tanstack/react-query";
+import axios from "axios";
+export const useGetAllIntegrations = () => {
+	const {
+		data: alleintegrasjoner,
```

Comment:
Er planen å gå for norsk her? apikallet er jo på norsk, så ser den, men ville stemt på allIntegrations uansett tror jeg. Eller alleIntegrasjoner om det må være på norsk.